mem0-ts/src/client/mem0.types.ts

```diff
@@ -31,6 +31,12 @@ export enum API_VERSION {
   V2 = "v2",
 }
 
+export enum Feedback {
+  POSITIVE = "POSITIVE",
+  NEGATIVE = "NEGATIVE",
+  NEUTRAL = "NEUTRAL",
```

Comment:
Do we have neutral as feedback? I believe this corresponds to single thumbs up on the platform, right? Might want to name it something else.